### ISSUE TITLE
questions about train datasets

### ISSUE THREAD

**Issue by hezl1592** [CVLAB-Unibo/Learning2AdaptForStereo#4]
thanks for the great work.
I have some questions about datasets for training.
1. the synthia dataset only have the depth map. when I use its provided focal length and baseline to get the disparity, it's data looks terrible. so I wanna know how you process these data.
2. I transform sparse depth info in kitti's video sequences, can I use these disparity data to train the net, for the data is really sparse when compared with the stereo datasets.
best wishes.
zilch

**Comment by AlessioTonioni** [comment]
For synthia we rendered our own sequences using the engine. The baseline and focal length of the virtual cameras were selected to closely match the one in kitti.

For Kitti you can either use the raw lidar maps or the cleaned depth maps available here: http://www.cvlibs.net/datasets/kitti/eval_depth.php?benchmark=depth_completion

The code should already take care of discarding not valid pixels (disparity = 0 or disparity > max_disparity) and computing the loss only on the valid ones.

**Comment by hezl1592** [comment]
Thanks for reply~
for the question 1, I mean that you mentioned you use this synthia datasets: http://synthia-dataset.net/  for training offline. So I wanna know the details about converting the depth map to disparity map, because I get the data seems wrong...
Also I wanna ask, the pretrain weights you provided [here](https://drive.google.com/open?id=1GwGxBOFx-NlUo9RAUgPlgPvaHCpGedlm) is just a dispNet weight? or it's a offline trained weight with WAD mode?
Best wishes!
zilch

**Comment by AlessioTonioni** [comment]
As for 1:
we have used the SYNTHIA VIDEO SEQUENCES (CVPR16) available here https://synthia-dataset.net/downloads/ and converted to disparity using the focal length and baseline of the simulated stero system. 
`depth = (baseline * focal)/disparity `

The pretrained weights are for Dispnet traind on FlyingThings3D

**Comment by hezl1592** [comment]
Thanks a lot~
best wishes
